demo/screencast-dsa-accel-config.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash -e
+
+PV='pv -qL'
+
+command()
+{
+  speed=$2
+  [ -z "$speed" ] && speed=10
+  echo "> $1" | $PV $speed
+  sh -c "$1"
+  echo | $PV $speed
+}
+
+out()
+{
+  speed=$2
+  [ -z "$speed" ] && speed=10
+  echo "$1" | $PV $speed
+  echo | $PV $speed
+}
+
+cleanup()
+{
+  clear
+  out 'Cleanup demo artifacts' 200
+  command 'kubectl delete pod dsa-accel-config-demo || true' 200
+  command 'kubectl delete -k https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/dsa_plugin || true' 200
+}
+
+record()
+{
+  clear
+  out 'Record this screencast'
+  command "asciinema rec -t 'Intel DSA Device Plugin for Kubernetes.'  Intel-DSA-Device-Plugin-for-Kubernetes-Demo.cast -c 'sh ./screencast-dsa-accel-config.sh play'" 300
+}
+
+screen1()
+{
+  clear
+  out "This screencast demonstrates deployment of the Intel DSA Plugin for Kubernetes"
+  out "Let's get started!"
+  out 'Check if DSA devices are available on the node:'
+  command 'accel-config list |grep "dev.*wq"'
+  sleep 2
+  command 'ls -la /dev/dsa/'
```

Comment:
what do you think about adding "check" kind of function that would verify that devices are present before trying to record? that would prevent somebody running script in wrong environment

Reply:
If there are no DSA devices on the node this script would fail I believe as ```ls -la /dev/dsa/``` command would not succeed.

Reply:
right,

```
Check if DSA devices are available on the node:

> ls -la /dev/dsa/
ls: cannot access '/dev/dsa/': No such file or directory
$
```

Reply:
(I commented out the accel-config check in my test ;)

Reply:
I think Sasha wanted to make this check before recording has started. That makes sense to me. I'll update the PR.

Reply:
@kad done, please review & merge, thanks.